api/v1beta2/ocirepository_types.go

```diff
@@ -78,6 +78,12 @@ type OCIRepositorySpec struct {
 	// +optional
 	SecretRef *meta.LocalObjectReference `json:"secretRef,omitempty"`
 
+	// Verify specifies which provider to use to check whether OCI image is authentic.
+	// Implements RFC-0003.
+	// https://github.com/fluxcd/flux2/tree/main/rfcs/0003-kubernetes-oci
```

Comment:
Please remove the RFC from here, the whole API is based on the RFC not just this field.

Reply:
done